src/Dialect/ONNX/ElementsAttr/StridesRange.hpp.inc

```diff
@@ -0,0 +1,61 @@
+//===------------------------ StridesRange.hpp.inc ------------------------===//
+//
+//===----------------------------------------------------------------------===//
+// StridesRange template implementations
+//===----------------------------------------------------------------------===//
+
+template <size_t N>
+inline auto StridesIterator<N>::operator++() -> StridesIterator<N> & {
+  ++(value.flattenedIndex);
+  for (auto axis = shape.size();;) {
```

Comment:
Why auto? its not consistent with the rest of the codebase.

Reply:
fixed